src/app.js

```diff
@@ -1 +1,44 @@
 'use strict';
+
+const readline = require('readline');
+const fs = require('fs');
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+let fileName = '';
+let fileContent = '';
+const reassignFileName = function(userInput) {
+  fileName = userInput;
+};
+const reassignFileContent = function(userInput) {
+  fileContent = userInput;
+};
+
+const askQuestion = (question, func) => {
+  return new Promise((resolve, reject) => {
+    rl.question(question, (userInput) => {
+      if (userInput) {
+        // eslint-disable-next-line no-console
+        console.log(userInput);
+        func(userInput);
+      }
+      resolve();
```

Comment:
The promise will be resolved in any case, even if the input field is empty and thus the file name will be empty in line 38. Could you please tell me where you check for the file name and content?